client/src/components/Markdown/Markdown.vue

```diff
@@ -188,7 +188,7 @@ export default {
                 this.workflows = config.workflows || {};
                 this.invocations = config.invocations || {};
                 this.loading = false;
-                this.workflowID = Object.keys(this.markdownConfig.workflows)[0];
+                this.workflowID = Object.keys(this.markdownConfig?.workflows ?? {})[0] ?? "";
```

Comment:
Should this maybe be undefined ? Also we seem to be calling `fetchWorkflowForInstanceId` and `getStoredWorkflowByInstanceId` with an empty string, that's probably also not quite right ?

Reply:
Yep, I blindly followed what was there as default. I'll fix it :+1: